src/components/Policy/EmptyState/Section.tsx

```diff
@@ -28,7 +28,6 @@ export const EmptyStateSection: React.FunctionComponent<EmptyStateSectionProps>
         className={ `${emptyStateClassName} ${props.className ? props.className : ''} ` }
         { ...getOuiaProps('Policy/EmptyState', props) }
     >
-        { props.icon && <EmptyStateIcon icon={ props.icon } color={ props.iconColor } /> }
```

Comment:
I think we still need to find the usages of `EmptyStateSection` (this component) and remove the icon/iconColor on the instances related to the [POL-469](https://issues.redhat.com/browse/POL-469) `Policy table` and `Trigger list table`

Reply:
the icon is only showing on the policy table.